src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/AbstractCloudFoundryDeployer.java

```diff
@@ -127,6 +130,21 @@ Path getApplication(AppDeploymentRequest request) {
 	}
 
 	/**
+	 * Return the Path to manifests.
+	 *
+	 */
+	Path getManifestsPath(AppDeploymentRequest request) {
+		try {
+			String manifestsPath = request.getDefinition().getProperties().get("manifests-path");
```

Comment:
This should definitely come from _deployment_ properties IMO, as this is a deployment concern. Why not use `request.getDeploymentProperties()`?